app/api/event_invoices.py

```diff
@@ -120,13 +131,13 @@ def before_create_object(self, data, view_kwargs):
             data['discount_code_id'] = discount_code.id
 
     view_kwargs = True
-    decorators = (is_admin,)
```

Comment:
But once @niranjan94 also told me the Event Invoices are managed by server admins and the generated invoices are sent to organizers by mail, so may be only server admins can see through APIs